Source: self-written
Upstream: yes
Reason: allow a datadir different from PREFIX/share

# HG changeset patch
# User Jakob Nixdorf <flocke@shadowice.org>
# Date 1455350699 -3600
#      Sat Feb 13 09:04:59 2016 +0100
# Node ID bb25842d07a36694d0afc27f0b609e62c3acf6f2
# Parent  74c350a85f7b95c297a27ffc49f032c72307e8b7
unix: allow defining a datadir different from PREFIX/share

diff -r 74c350a85f7b -r bb25842d07a3 texstudio.pro
--- a/texstudio.pro	Fri Feb 12 20:42:36 2016 +0100
+++ b/texstudio.pro	Sat Feb 13 09:04:59 2016 +0100
@@ -305,22 +305,23 @@
 # ###############################
 unix:!macx {
     isEmpty( PREFIX ):PREFIX = /usr
+    isEmpty( DATADIR ):DATADIR = $${PREFIX}/share
     DEFINES += PREFIX=\\\"$${PREFIX}\\\"
     target.path = $${PREFIX}/bin
-    utilities.path = $${PREFIX}/share/texstudio
-    manual.path = $${PREFIX}/share/texstudio
+    utilities.path = $${DATADIR}/texstudio
+    manual.path = $${DATADIR}/texstudio
     utilities.files = utilities/texstudio16x16.png \
         utilities/texstudio22x22.png \
         utilities/texstudio32x32.png \
         utilities/texstudio48x48.png \
         utilities/texstudio64x64.png \
         utilities/texstudio128x128.png
-    applicationmenu.path = $${PREFIX}/share/applications
+    applicationmenu.path = $${DATADIR}/applications
     applicationmenu.files = utilities/texstudio.desktop
-    icon.path = $${PREFIX}/share/icons/hicolor/scalable/apps
+    icon.path = $${DATADIR}/icons/hicolor/scalable/apps
     icon.files = utilities/texstudio.svg
     isEmpty(NO_APPDATA) {
-      appdata.path = /usr/share/appdata
+      appdata.path = $${DATADIR}/appdata
       appdata.files = utilities/texstudio.appdata.xml
       INSTALLS += appdata
     }
